components/RepoListView.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { FlatList, StyleSheet } from 'react-native'
+import { ListItemView } from './ListItemView'
+
+interface Props {
+    data: any[];
+    onItemPress: (itemId: number) => void;
+}
+
+export const RepoListView = ({ data, onItemPress }: Props) => {
+    return (
+        <FlatList
+            data={data}
+            style={styles.list}
+            renderItem={({ item }) =>
+                <ListItemView
+                    key={item.key}
```

Comment:
This piece ```key={item.key}``` can be change by ```key={item.id}```, because it is a field that never change in the user,  that up to you

Reply:
I dont know what kind of data will be retrieve this will be changing in the next step. This is part of next step.

Reply:
sure, but you can use the id field if you want to be sure that you are using an unchangeable datum(id never chance for a user)